rebar.config.script

```diff
@@ -1,14 +1,8 @@
 IsRebar3 = erlang:function_exported(rebar3, main, 1),
-IsNoSnappy = case os:getenv("KAFKA_PROTOCOL_NO_SNAPPY") of
-               "1" -> true;
-               _   -> false
-             end,
-case {IsNoSnappy, IsRebar3} of
-  {true, _} ->
-    lists:keyreplace(deps, 1, CONFIG, {deps, []});
-  {false, true} ->
+case IsRebar3 of
```

Comment:
this is to downgrade dependency tuple from rebar3 format to rebar format.
not to delete the dependency